CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+### [0.3.1](https://github.com/Human-Connection/Human-Connection/compare/v0.3.0...v0.3.1) (2020-02-04)
```

Comment:
Why this change?

Remove this file from the PR and update the changelog once we merged everything in, I would say?